controller_manager/src/controller_manager.cpp

```diff
@@ -286,7 +286,13 @@ ControllerManager::ControllerManager(
       "[Deprecated] Passing the robot description parameter directly to the control_manager node "
       "is deprecated. Use '~/robot_description' topic from 'robot_state_publisher' instead.");
     init_resource_manager(robot_description_);
-    init_services();
+    if (resource_manager_->are_components_initialized())
+    {
+      RCLCPP_FATAL(
```

Comment:
```suggestion
      RCLCPP_WARN(
```

I'm not sure if this is a FATAL level. I think it needs to be a warning as the system will continue to work

Reply:
Oh, yes, you are right! thanks!